supabase/migrations/20250320131019_create_projects_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX projects_fqdn_idx ON public.projects USING fqdn;
```

Comment:
```suggestion
CREATE INDEX projects_fqdn_idx ON public.projects (fqdn);
```

Le `USING` c'est pour choisir la méthode d'indexation (ex: `USING GIN`)